sample-app/src/components/FilterSearch.tsx

```diff
@@ -37,6 +41,7 @@ export default function FilterSearch ({
   const searchParamFields = searchFields.map((searchField) => {
     return { ...searchField, fetchEntities: false }
   });
+  const cssClasses = builtInCssClasses;
```

Comment:
We'll need to merge the custom and builtInClasses here.

Reply:
yep, I hadn't gotten around to merging the `useComposedCssClasses` hook yet (although it's ready for review again)